src/handlers/comment/handlers/index.ts

```diff
@@ -75,6 +76,16 @@ export const issueCreatedCallback = async (): Promise<void> => {
     if (priorityLabels.length === 0 && priorityLabelConfigs.length > 0) await createLabel(priorityLabelConfigs[0].name);
     await addLabelToIssue(timeLabelConfigs[0].name);
     await addLabelToIssue(priorityLabelConfigs[0].name);
+
+    const targetPriceLabel = getTargetPriceLabel(timeLabelConfigs[0].name, priorityLabelConfigs[0].name);
+    if (targetPriceLabel) {
+      const exist = await getLabel(targetPriceLabel);
+      if (!exist) {
+        await createLabel(targetPriceLabel, "price");
+      }
+      await addLabelToIssue(targetPriceLabel);
+    }
+
```

Comment:
would be better to remove duplication use of `timeLabelConfigs[0].name` and `priorityLabelConfigs[0].name`